provisioner/docker/docker-hadoop.sh

```diff
@@ -284,12 +293,19 @@ log() {
 }
 
 configure-nexus() {
+    get_nodes
     for node in ${NODES[*]}; do
         docker exec $node bash -c "cd /bigtop-home; ./gradlew -PNEXUS_URL=$1 configure-nexus"
     done
     wait
 }
 
+get_nodes() {
+    if [ -n "$PROVISION_ID" ]; then
+        # shellcheck disable=SC2207
+        NODES=(`$DOCKER_COMPOSE_CMD -p $PROVISION_ID ps -q`)
+    fi
```

Comment:
Shall we error out earlier and explicitly log what's the problem when getting nodes?

Reply:
I am open to any suggestion, I have basically wrapped what was there in a function.. What do you have in mind? I can incorporate it in the change :)

Reply:
My bad for not being clear. I was thinking that for the else case (PROVISION_ID is not set), we can error out earlier with a clear log message instead of being silence . But this is a minor comment as we didn't do this kind of early error out in other place, plus you're just wrapping what originally written ;)

Reply:
@evans-ye ah yes definitely! I can follow up with another commit :)